tests/integ/preds.txt

```diff
@@ -25,3 +25,7 @@ thingy is 200 and val is 100 # false
 {"Jack" "Joe" "Jill"} contains name # true
 {400 500} contains status # true
 {"east-db-001" "east-foo-002"} contains server # false
+errors contains anyof {'cpu load' 'foo'} # true
+errors contains anyof {'foo' 'bar'} # false
```

Comment:
Have you thought about what happens with `error not contains anyof` 🤯 ?

Reply:
Good point, will try next week 😅

Reply:
So it works, but the syntax is `not error contains anyof` 🙂

Reply:
Cool. What happens if you invert? (i.e `error not contains anyof`)?

Reply:
It throws an error saying that the predicate is invalid with an explanation like `Syntax error with not at line 1, col 8`

Reply:
Ok, this should be good for our purposes then.